arionBackend/tests/api/test_event_type.py

```diff
@@ -0,0 +1,87 @@
+"""
+This module holds the test cases for the event type API module.
+"""
+import json
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from arionBackend.models.event_type import EventType
+from arionBackend.models.hierarchy import Hierarchy
+from arionBackend.models.query import Query
+
+
+class EventTypeTestCase(APITestCase):
+	"""
+	This class contains the tests for the event type API.
+	"""
+
+	@classmethod
+	def setUpTestData(cls):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method "setUpTestData" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS100)